endpoints/openrtb2/video_auction.go

```diff
@@ -140,6 +140,14 @@ func (deps *endpointDeps) VideoAuctionEndpoint(w http.ResponseWriter, r *http.Re
 	//create full open rtb req from full video request
 	mergeData(videoBidReq, bidReq)
 
+	//if Device.UA is not present in request body, init it with user-agent from request header if it's present
+	if videoBidReq.Device.UA == "" {
+		userAgent := r.Header.Get("User-Agent")
```

Comment:
You could probably just write `videoBidReq.Device.UA = r.Header.Get("User-Agent")` as nothing really changes if the header value is empty and the device ua is already empty.

Reply:
Yes!